backend/dataall/modules/datapipelines/api/resolvers.py

```diff
@@ -237,23 +221,15 @@ def get_creds(context: Context, source, DataPipelineUri: str = None):
         )
 
 
-def _delete_repository(
-    target_uri, accountid, cdk_role_arn, region, repo_name
-):
-    context = get_context()
-    with context.db_engine.scoped_session() as session:
-        task = models.Task(
-            targetUri=target_uri,
-            action='repo.datapipeline.delete',
-            payload={
-                'accountid': accountid,
-                'region': region,
-                'cdk_role_arn': cdk_role_arn,
-                'repo_name': repo_name,
-            },
-        )
-        session.add(task)
-    Worker.queue(context.db_engine, [task.taskUri])
```

Comment:
Not sure if this one should be removed. `Worker#process` runs synchronously, but it was async.

Reply:
I readded the function in the resolver the way it was. I also readded the codecommit handler which now handles only this deletion task